### PR TITLE
Implement a GUI for WILDCARD_SENDERS

### LINKED ISSUE #2372: Create a GUI for WILDCARD_SENDERS
It would be nice to have a GUI, as discussed in #1904

### PR DESCRIPTION
**Pull request by nextgens**
## What type of PR?

Feature

## What does this PR do?

- Implement a GUI for WILDCARD_SENDERS

### Related issue(s)
- closes #2372

## Prerequisites
Before we can consider review and merge, please make sure the following list is done and checked.
If an entry in not applicable, you can check it or remove it from the list.

- [x] In case of feature or enhancement: documentation updated accordingly
- [x] Unless it's docs or a minor change: add [changelog](https://mailu.io/master/contributors/workflow.html#changelog) entry file.

### CODE REVIEW

**Comment by nextgens** [comment]
bors try

**Comment by nextgens** [comment]
bors try

File: core/admin/mailu/internal/views/postfix.py [hunk 1]
```diff
@@ -145,8 +145,9 @@ def postfix_sender_login(sender):
     localpart = localpart[:next((i for i, ch in enumerate(localpart) if ch in flask.current_app.config.get('RECIPIENT_DELIMITER')), None)]
     destinations = models.Email.resolve_destination(localpart, domain_name, True) or []
     destinations.extend(wildcard_senders)
+    destinations.extend(i[0] for i in models.User.query.filter_by(allow_spoofing=True).with_entities(models.User.email).all())
     if destinations:
-        return flask.jsonify(",".join(idna_encode(destinations)))
+        return flask.jsonify(",".join(idna_encode(list(set(destinations)))))
     return flask.abort(404)
```
Comment (ghostwheel42) [inline core/admin/mailu/internal/views/postfix.py:151]: You don't need to use `list()` here.
What about this for the function:
```
def postfix_sender_login(sender):
    wildcard_senders = {s for s in flask.current_app.config.get('WILDCARD_SENDERS', '').lower().replace(' ', '').split(',') if s}
    localpart, domain_name = models.Email.resolve_domain(sender)
    if localpart is None:
        return flask.jsonify(",".join(wildcard_senders)) if wildcard_senders else flask.abort(404)
    destinations = set(models.Email.resolve_destination(localpart, domain_name, True) or [])
    destinations.update(wildcard_senders)
    destinations.update(a[0] for a in models.User.query.filter_by(allow_spoofing=True).with_entities(models.User.email))
    if destinations:
        return flask.jsonify(",".join(idna_encode(destinations)))
```

**Review by ghostwheel42 — CHANGES_REQUESTED** [review]
LGTM. I'd like to use more sets in postfix_sender_login. Okay?

File: core/admin/mailu/internal/views/postfix.py (hunk 1 above)
Comment (nextgens) [reply core/admin/mailu/internal/views/postfix.py]: You're missing a line, but yes, sure. Done in 19bd9362

**Comment by nextgens** [comment]
bors try

**Review by ghostwheel42 — CHANGES_REQUESTED** [review]
There's a Problem with alembic.
Maybe you used some local revision?

File: core/admin/migrations/versions/7ac252f2bbbf_.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+""" Add user.allow_spoofing
```
[the remark this answers is not in the document]
Comment (nextgens) [reply core/admin/migrations/versions/7ac252f2bbbf_.py:1]: Yeah that was created on another PR... so I've merged master and now this should be gone.

**Review by ghostwheel42 — APPROVED** [review]
LGTM

**Pull request merged**